release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -799,6 +799,28 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-web-console-developer-perspective-bug-fixes"]
 ==== Web console (Developer perspective)
 
+* Before this update, resources in the *Developer* perspective of the web console had invalid links to details about that resource. This update resolves the issue. It creates valid links so that users can access resource details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2000651[BZ#2000651])
+
+* Before this update, you could only specify a subject in the `SinkBinding` form by label, not by name. With this update, you can use a drop-down list to select whether to specify a subject by name or label. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002266[BZ#2002266])
+
+* Before this update, the web terminal icon was available in the web console's banner head only if you installed the Web Terminal Operator in the `openShift-operators` namespace. With this update, the terminal icon is available regardless of the namespace where you install the Web Terminal Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2006329[2006329])
+
+* Before this update, the service binding connector did not appear in topology if you used a `resource` property instead of a `kind` property to define a `ServiceBinding` custom resource (CR). This update resolves the issue by reading the CR's `resource` property to display the connector on the topology. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013545[BZ#2013545])
+
+* Before this update, the name input fields used a complex and recursive regular expression to validate user inputs. This regular expression made name detection very slow and often caused errors. This update resolves the issue by optimizing the regular expression and avoiding recursive matching. Now, name detection is fast and does not cause errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014497[BZ#2014497])
+
+* Before this update, feature flag gating was missing from some extensions contributed by the knative plugin. Although this issue didn't affect what was displayed, these extensions ran unnecessarily, even if the serverless operator was not installed. This update resolves the issue by adding feature flag gating to the extensions where it was missing. Now, the extensions do not run unnecessarily. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2016438[BZ#2016438])
```

Comment:
Second sentence contraction, `didn't`. I don't think the contraction is bad here. However, I don't notice many contractions throughout these web console updates. For consistency, I would simply avoid using contractions unless you do so frequently throughout the updates.

Reply:
Updated. Thanks.